<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicList</class>
 <widget class="QWidget" name="MusicList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="styleSheet">
              <string notr="true">font: 14pt &quot;Sitka Small&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>请输入要查找的歌曲名（点击后即缓存到本地目录）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="musicplayer.qrc">
               <normaloff>:/final_work/images/search.png</normaloff>:/final_work/images/search.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 81 10pt &quot;汉仪雅酷黑 85W&quot;;</string>
           </property>
           <property name="text">
            <string>在线音乐</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>700</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>450</width>
             <height>700</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 81 10pt &quot;汉仪雅酷黑 85W&quot;;</string>
         </property>
         <property name="text">
          <string>本地音乐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>10000</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="musicplayer.qrc"/>
 </resources>
 <connections/>
</ui>
